lib/secure_upload/property_format.c

```diff
@@ -0,0 +1,42 @@
+#include "property_format.h"
+#include "upload.h"
+
+// list that tells for each property its format string
+const char *format_property[] = {
+    "%d", // PROPERTY_HEARTBEAT
+    "%u", // CO2_CONCENTRATION
```

Comment:
The bottom three properties measured by the SCD41 should ultimately not be in the generic firmware but in firmware repo of https://github.com/energietransitie/twutility-twomes-co2-bt-occupancy-monitor-firmware

Reply:
Fixed in https://github.com/energietransitie/twomes-generic-esp-firmware/commit/cfa8e2cbd446a736b5f27a06d15798c9525fe762